app/controllers/assignments_controller.rb

```diff
@@ -202,6 +202,9 @@ def edit
                                    .where.not(id: @assignment.id)
                                    .order(:id)
     @sections = Section.all
+    @section_due_dates = SectionDueDate.joins(:section)
```

Comment:
It seems like this will be nil if the assignment has no section due dates (e.g., when the assignment is first created), leading this to be empty.